.travis.yml

```diff
@@ -61,13 +62,15 @@ notifications:
 
 before_deploy:
   - export GEOTRELLIS_VERSION_SUFFIX="-${TRAVIS_COMMIT:0:7}"
+  - ./sbt unidoc
 
 deploy:
   - provider: script
     script: .travis/scaladocs.sh
     skip_cleanup: true
     on:
       repo: locationtech/geotrellis
-      branch: master
-      jdk: oraclejdk8
+      branch: fixup/awf/deploy-scaladoc-latest
+      condition: "$RUN_SET = 'scaladoc'"
```

Comment:
The addition of this Travis deploy condition and the additional `RUN_SET=scaladoc` above ensures we only trigger the deploy once per Travis build and also makes it clear about when this scaladoc push happens -- in its own run rather than tacked on to another run that is also doing other things.